vars/pipelineApmUI.groovy

```diff
@@ -309,7 +309,7 @@ def checkoutES(){
 */
 def checkoutKibana(){
 //  useCache('source'){
-    gitCheckout(basedir: "${BASE_DIR}", branch: params.branch_specifier,
+    gitCheckout(basedir: "${BASE_DIR}", branch: args.branch_specifier,
```

Comment:
```suggestion
    gitCheckout(basedir: "${BASE_DIR}", branch: params.branch_specifier,
```

Reply:
I've deleted this pipeline it is not used.